library/components/TTooltip.vue

```diff
@@ -33,53 +38,58 @@ export default {
       type: Boolean,
       default: false,
-    }
+    },
   },
   data: () => ({
-    positions: { arrow: {}, tooltip: {} },
-    isVisible: this.isOpen,
+    styles: { arrow: {}, tooltip: {} },
+    isVisible: false,
+    isTooltipFocused: false, // If cursor is on tooltip.
   }),
   methods: {
     placeTooltip() {
       // Calculate and place both tooltip and arrow.
       const triggerElement = this.$refs.trigger;
       const tooltipElement = this.$refs.tooltip;
 
-      const positions = { ...this.positions };
+      const styles = { ...this.styles };
       if (triggerElement && tooltipElement) {
-        const position = triggerElement.getBoundingClientRect();
         if (this.position === "top") {
-          positions.tooltip.top = position.top - (tooltipElement.clientHeight + 10) + "px";
-          positions.tooltip.left = position.left - tooltipElement.clientWidth / 2 + "px";
-          positions.arrow.top = position.top - 20 + "px";
-          positions.arrow.left = position.left + "px";
+          styles.tooltip.bottom = triggerElement.clientHeight + "px";
+          styles.tooltip.left = "-25px";
+          styles.tooltip.paddingBottom = "10px";
+          styles.arrow.bottom = "-6px";
+          styles.arrow.left = "25px";
         } else if (this.position === "bottom") {
-          positions.tooltip.top = position.top + triggerElement.clientHeight + 15 + "px";
-          positions.tooltip.left = position.left - tooltipElement.clientWidth / 2 + "px";
-          positions.arrow.top = position.top + triggerElement.clientHeight + 10 + "px";
-          positions.arrow.left = position.left + "px";
+          styles.tooltip.top = triggerElement.clientHeight + "px";
+          styles.tooltip.left = "-25px";
+          styles.tooltip.paddingTop = "10px";
+          styles.arrow.top = "-6px";
+          styles.arrow.left = "25px";
         } else if (this.position === "left") {
-          positions.tooltip.top = position.top - tooltipElement.clientHeight / 2 + "px";
-          positions.tooltip.left = position.left - tooltipElement.clientWidth - 15 + "px";
-          positions.arrow.top = position.top + "px";
-          positions.arrow.left = position.left - 25 + "px";
+          styles.tooltip.top = "0px";
+          styles.tooltip.left = "-" + (tooltipElement.clientWidth + 10) + "px";
+          styles.tooltip.paddingRight = "10px";
+          styles.arrow.right = "-6px";
+          styles.arrow.top = "10px";
         } else {
-          positions.tooltip.top = position.top - tooltipElement.clientHeight / 2 + "px";
-          positions.tooltip.left = position.left + triggerElement.clientWidth + 15 + "px";
-          positions.arrow.top = position.top + "px";
-          positions.arrow.left = position.left + triggerElement.clientWidth + 10 + "px";
+          styles.tooltip.top = "0px";
+          styles.tooltip.left = triggerElement.clientWidth + "px";
+          styles.tooltip.paddingLeft = "10px";
```

Comment:
With a bit of tweaking and using paddings, I was able to get rid of debounce and make tooltips hoverable.

Reply:
amazing!